src/feedback/feedback.tpl

```diff
@@ -1,7 +1,9 @@
-<div id="{{id}}" class="feedback feedback-{{level}} {{#if popup}}popup{{/if}}">
+<div id="{{id}}" class="feedback feedback-{{level}} {{#if popup}}popup{{/if}}"
+     role="dialog" aria-describedby="{{dompurify msg}}" >
     <span class="icon-{{level}}"></span>
-    <div>
+    <div aria-live="{{dompurify msg}}">
```

Comment:
Seems `{{dompurify msg}}` isn't in the list of possible values, which can be accepted by the aria-live attribute, they are: off, polite and assertive https://www.w3.org/TR/wai-aria-1.1/#aria-live